data-pipeline/src/data_pipeline/pipelines/genes.py

```diff
@@ -62,6 +65,10 @@
     "/genes/genes_grch37_base.ht",
     {"gencode_path": pipeline.get_task("download_gencode_v19_gtf"), "hgnc_path": pipeline.get_task("download_hgnc")},
     {"reference_genome": "GRCh37"},
+    # TODO:FIXME:
+    # 6th positional argument is subsettable
+    subsettable=True,
+    # True,
 )
 
```

Comment:
Starting to wonder if instead of passing subset flags into each pipeline step, we could conditionally insert a new task in front of main task that does the subsetting before passing it on to the main task.